text/0000-replace-blacklist-whitelist.md

```diff
@@ -0,0 +1,70 @@
+- Start Date: 2020-06-15
+- Relevant Team(s): Ember CLI, Learning
+- RFC PR: (after opening the RFC PR, update this with a link to it and update the file name)
+- Tracking: (leave this empty)
+
+# Replace terms blacklist & whitelist in Ember CLI
+
+## Summary
+
+Ember.js prides itself (rightly) on being an inclusive framework. To further improve on this, we should remove the terms "blacklist" and "whitelist" with more neutral replacements.
+
+## Motivation
+
+The terms "blacklist" and "whitelist" are considered racially insenstive. These terms are used in Ember CLI for rather advanced functionality, which should make providing a better alternative naming and deprecating the current one rather easy.
```

Comment:
This seems to be making a specific claim (that the terms are racially motivated), can you cross link reference material to make it clear this isn't just an assumption being made.

Reply:
I have added some examples and references, hope that works better!

Reply:
There is a lot of material that is available about the subliminal messaging that is pervasive in American culture whose underlying message is "white is good" and that has made its way into a lot of things as "the norm". 

While the original developer who first wrote "whitelist" and "blacklist" was perhaps not a racist (but rather using the words that had been made social norms by a white majority), we have enough public awareness to choose an elevated, more inclusive way of doing things for Ember projects.

Reply:
> There is a lot of material that is available about the subliminal messaging that is pervasive in American culture whose underlying message is "white is good" and that has made its way into a lot of things as "the norm".

Awesome, that was my assumption when I made my comment above. It should be super easy to satisfy my request. 😃

Reply:
> While the original developer who first wrote "whitelist" and "blacklist" was perhaps not a racist (but rather using the words that had been made social norms by a white majority), we have enough public awareness to choose an elevated, more inclusive way of doing things for Ember projects.

Not sure what this was replying to at all.